File: server/icecreamapp/src/main/java/com/ssafy/icecreamapp/model/service/ReviewService.java
package com.ssafy.icecreamapp.model.service;

import com.ssafy.icecreamapp.model.dto.Review;
import com.ssafy.icecreamapp.model.dto.ReviewCon;
import com.ssafy.icecreamapp.model.dto.request.InitReview;

import java.util.List;

public interface ReviewService {

    int addReview(InitReview initReview);
    List<Review> selectReviews(ReviewCon reviewCon);
}

File: android/app/src/main/java/com/ssafy/icepop/data/remote/ReviewService.kt
package com.ssafy.icepop.data.remote

import com.ssafy.icepop.data.model.dto.IceCreamOrderReview
import com.ssafy.icepop.data.model.dto.request.ReviewListRequest
import com.ssafy.icepop.data.model.dto.request.ReviewPostRequest
import retrofit2.http.Body
import retrofit2.http.POST

interface ReviewService {
    @POST("review/getReviews")
    suspend fun getReviewList(@Body reviewListRequest: ReviewListRequest) : List<IceCreamOrderReview>

    @POST("review/addReview")
    suspend fun postReview(@Body reviewPostRequest: ReviewPostRequest) : String
}